ingestion/functions/parsing/peru/peru_test2.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import json
+import os
+import tempfile
+import unittest
+from datetime import date
+from peru import peru
+
+_SOURCE_ID = "place_holder"
+_SOURCE_URL = "place_holder"
+
+
+class PeruTest(unittest.TestCase):
+    def setUp(self):
+        # Default of 1500 is not enough to show diffs when there is one.
+        self.maxDiff = 5000
+
+    def test_parse(self):
```

Comment:
different tests shoudl be in the same file, just different methods within the PeruTest class. Do not use peru_test.py and peru_test2.py please.